data/projects/projects.router.js

```diff
@@ -0,0 +1,146 @@
+const express = require("express");
+const db = require("./projects.model");
+
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  db.getProjects()
+    .then(projects => {
+      res.status(200).json(projects);
```

Comment:
put a .foreach with an if statement in here to get true or false instead of 1 or 0